books/history/bake

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+require_relative '../../bake_helper'
+
+gemfile do
+  gem 'openstax_kitchen', \
+      ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : KITCHEN_VERSIONS[:history]
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :history) do |doc|
+  include Kitchen::Directions
+
+  book = doc.book
+  metadata = book.metadata
+
+  # Some stuff just goes away
+  book.search('cnx-pi').trash
+
+  BakePreface.v1(book: book, title_element: 'h1')
+  BakeChapterIntroductions.v1(book: book)
+  BakeChapterTitle.v1(book: book)
+
+  BakeAutotitledNotes.v1(
+    book: book,
+    classes: %w[click-and-explore my-story americana defining-american]
+  )
+
+  answer_key = BookAnswerKeyContainer.v1(book: book, solutions_plural: false)
+
+  book.chapters.each do |chapter|
+    BakeLearningObjectives.v1(chapter: chapter)
+    BakeNonIntroductionPages.v1(chapter: chapter)
+    BakeChapterGlossary.v1(chapter: chapter, metadata_source: metadata)
+    BakeChapterSummary.v1(chapter: chapter, metadata_source: metadata)
+
+    eoc_sections = %w[review-questions critical-thinking]
+
+    chapter.search(eoc_sections.prefix('section.')).exercises.each do |exercise|
+      if exercise.parent.has_class?('review-questions')
+        BakeNumberedExercise.v1(
+          exercise: exercise,
+          number: exercise.count_in(:chapter),
+          suppress_solution_if: :even?,
+          note_suppressed_solutions: true
+        )
+      else
+        BakeNumberedExercise.v1(exercise: exercise, number: exercise.count_in(:chapter))
+      end
+    end
+
+    chapter.figures('$:not(.unnumbered)').each do |figure|
+      BakeFigure.v1(figure: figure,
+                    number: "#{chapter.count_in(:book)}.#{figure.count_in(:chapter)}")
+    end
+
```

Comment:
If there are unnumbered figures in the book we should use also `BakeUnnumberedFigure.v1(book: book)` method. This is something that differs from old recipe, but Larissa wanted to use it everywhere where such figurs appear.

Also we can define figure more precisely

![image](https://user-images.githubusercontent.com/40228854/142367790-167eab9a-0111-4ab3-aff1-c2999a58af43.png)

This method excludes also subfigures, but maybe it's not necessery for this book. So I leave this up to you :wink: